### ISSUE TITLE
Allow option to disable initial alphabet in byte_level_bpe.py

### ISSUE THREAD

**Issue by MartensCedric** [huggingface/tokenizers#1220]
In the code from `train_from_iterator`

```
  def train_from_iterator(
        self,
        iterator: Union[Iterator[str], Iterator[Iterator[str]]],
        vocab_size: int = 30000,
        min_frequency: int = 2,
        show_progress: bool = True,
        special_tokens: List[Union[str, AddedToken]] = [],
        length: Optional[int] = None,
    ):
        """Train the model using the given iterator"""

        trainer = trainers.BpeTrainer(
            vocab_size=vocab_size,
            min_frequency=min_frequency,
            show_progress=show_progress,
            special_tokens=special_tokens,
            initial_alphabet=pre_tokenizers.ByteLevel.alphabet(),  # <-- initial alphabet is forced here.
        )
        self._tokenizer.train_from_iterator(
            iterator,
            trainer=trainer,
            length=length,
        )
```

In my case I am working with an extremely non-standard alphabet (i.e ATCG sequences). Then, when I looked at the vocabulary, I see this:

 'Ď', 'TAATC', 'CCCT', 'AACG', 'ç', '_', 'GGGTC', 'TGCTTTTTCG', 'ķ', 'z', 'GTTGC',
 
 
 I don't want  'Ď', 'ç', '_',  'ķ', 'z', in my vocabulary. It would be nice to disallow the initial alphabet through `train` and `train_from_iterator`

**Comment by Narsil** [comment]
I am not sure which code you're referring to but 

Can't you simply break it into parts ?
 
```python
tokenizer = #.. Some code to create the tokenizer

    trainer = trainers.BpeTrainer(
            vocab_size=vocab_size,
            min_frequency=min_frequency,
            show_progress=show_progress,
            special_tokens=special_tokens,
            initial_alphabet=pre_tokenizers.ByteLevel.alphabet(),  # <-- initial alphabet is forced here.
        )
tokenizer.train_from_iterator(
            iterator,
            trainer=trainer,
            length=length,
        )
```

**Comment by MartensCedric** [comment]
I am referring to the code that I added an arrow 

```
initial_alphabet=pre_tokenizers.ByteLevel.alphabet(),  # <-- initial alphabet is forced here.
```

**Comment by Narsil** [comment]
It's not forced... it's specified in here, you can modify it at will here..

just modify the line if you want something else, no ?

**Comment by owos** [comment]
@Narsil please is there a way to set some seed tokens that must be learnt when training a new tokenizer?
